apps/daimo-mobile/src/view/screen/HomeScreen.tsx

```diff
@@ -281,6 +283,47 @@ function IconButton({
   );
 }
 
+let deepLinkInitialised = false;
+
+/** Handle incoming app deep links. */
+function useInitNavLinks() {
```

Comment:
Moved this here because `nav.ts` is used in a test, and anything that is upstream of `react-native-nfc-manager` doesn't work in tests

Reply:
maybe move to `deeplink.ts` instead